users/src/activity/index.ts

```diff
@@ -0,0 +1,31 @@
+import { Router } from "express";
+import { Activity } from "../entities/Activity";
+
+
+const router = Router()
+
+router.post("/", async (req, res) => {
+    const em = req.orm.em.fork();
+    
+    const uid = req.firebaseToken.uid;
+    const body = req.body;
+
+    const activity = await em.upsert(Activity,
+            { uid: uid, questionId: body.questionId })
+
+    await em.persistAndFlush(activity);
+
+    res.send({ success: true });
```

Comment:
I think Siva was hoping for a standard API response type (in the tele grp) but I'm good with this too. Might wanna hash this out in the tele grp so its clear when frontend can depend on the status message style, and when they need to write their own.

Reply:
Resolved